authors/apps/comments/renderers.py

```diff
@@ -15,3 +16,15 @@ def render(self, data, media_type=None, renderer_context=None):
             'likes': data,
             "likes count": len(data)
         })
+
+
+class CommentJSONRenderer(JSONRenderer):
+    charset = 'utf-8'
+
+    def render(self, data, media_type=None, renderer_context=None):
+
+        if type(data) != ReturnList:
```

Comment:
@moseskamira I would like to know what happens when the type(data) is ReturnList